Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -206,18 +206,6 @@ - (void)configureAppearance {
     }
 }
 
-- (void)setAccessibilityLabel:(NSString *)accessibilityLabel {
-    self.titleLabel.accessibilityLabel = accessibilityLabel;
-}
-
-- (void)setAccessibilityTraits:(UIAccessibilityTraits)accessibilityTraits {
-    self.titleLabel.accessibilityTraits = accessibilityTraits;
-}
-
-- (UIAccessibilityTraits)accessibilityTraits {
-    return self.titleLabel.accessibilityTraits;
-}
-
```

Comment:
We used to pass these values down to the title label as this was required to make FSCalendar play nice. Evidently this is no longer required, so was actually breaking accessibility 🤦